demos/clock/src/question/components/utils/shapeDrawers.js

```diff
@@ -0,0 +1,84 @@
+import * as d3 from "d3";
+import { CLOCK_CENTER } from "../../constants";
+import { getPointOnCircle } from "./pointFinders";
+
+/* CLOCK_CENTER is the default value 
+for cx,cy in a circle, and x1, y1 in a hand
+(the place from which circles will originate
+and to which the clock hands will be anchored)
+*/
```

Comment:
Pls. wrap this in a proper comments
```suggestion
/**
 * CLOCK_CENTER is the default value
 * for cx,cy in a circle, and x1, y1 in a hand
 * (the place from which circles will originate
 * and to which the clock hands will be anchored)
 */
```